src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -24,7 +24,7 @@ static ucs_config_field_t uct_rdmacm_md_config_table[] = {
 };
 
 static ucs_config_field_t uct_rdmacm_cm_config_table[] = {
-    {"", "", NULL,
+    {"RDMA_CM_", "", NULL,
```

Comment:
CM_
pls fix in TCP as well

Reply:
you mean without RDMA_ ? just CM_ ?
why?

Reply:
it should be just CM_ because it's parent's table prefix
pls see other transports as example

Reply:
but then the 'inherits' part is wrong since the parent doesn't have it:
https://github.com/openucx/ucx/blob/master/src/uct/base/uct_cm.c#L18 (where is this prefix set?)

```
#
# Allow using an address that is already in use.
#
# syntax:    <y|n>
# inherits:  UCX_CM_REUSEADDR
#
UCX_RDMA_CM_REUSEADDR=n
```

Reply:
parent does not need to have it since parsing starts from child